cmd/spec/path.yaml

```diff
@@ -112,6 +112,27 @@ paths:
                   $ref: '#/components/schemas/v1.SourceResponse'
         '500':
           $ref: "#/components/responses/InternalError"
+  /sources/{ID}/account_identity:
+    get:
+        parameters:
+        - in: path
+          name: ID
+          schema:
+            type: integer
+            format: int64
+          required: true
+          description: 'Source ID from Sources Database'
+        responses:
+          '200':
+            description: Return on success.
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/v1.AccountIDTypeResponse'
+          '404':
+            $ref: "#/components/responses/NotFound"
+          '500':
+            $ref: "#/components/responses/InternalError"
```

Comment:
Maybe it you could drop full description in the OpenAPI spec about why this is needed, who needs it and that it might actually be deprecated in the future. Let's be honest to our API users from the day one :-)